### PR TITLE
docs: add doc readme for pouch

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

**1.Describe what this PR did**

This PR adds a REAME.md in the docs directory. We can treat this file as an brief and clear introduction of architecture of document's organization.

We can treat them as the following contents:

* Features
* Get Started
* Commandline
* API
    * HTTP API
    * HTTP API changelog
    * CRI grpc API
    * CRI grpc changelog
* Contributions
* Test Guide
* Underlying Tech
* Static Files

**2.Does this pull request fix one issue?** 

NONE

**3.Describe how you did it**
NONE

**4.Describe how to verify it**
NONE

**5.Special notes for reviews**
NONE

### CODE REVIEW

**Comment by allencloud** [comment]
Could you please take a look of this?

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* [Underlying Tech](#underlying-tech)
+* [Static Files](#static-files)
+
+## Features
+
+[Features](features) contains all features of Pouch. To enable a satisfying environment for user's applications, Pouch provides plenty of features, such as strong-isolation technology including hyper-based containers and lxcfs-base resource view isolation, rich container, P2P image distribution and so on. For more aspects, TLS-based security, prometheus supporting and more features also help Pouch be adoptive for enterprises. In addition, in all the document we illustrate what is the best scenario of the corresponding feature. 
+
+## Get Started
+
+[Get Started](../INSTALLATION.md) is exactly what you need if you would give Pouch a try. This document includes what is the prerequisites, how to install Pouch and how to experience Pouch.
```
Comment (Letty5411) [inline docs/README.md]: what is the prerequisites -> are

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Commandline
+
+For almost all of users, [commandline](./commandline) is the first reference you may need. Document in directory commandline is about command detailed usage of pouch cli. You can get introductions, synopsis, examples, options about command. Last but not least, pouch can guarantee commandline docs is strongly consistent with pouch cli's source code. What's more, all commandline docs are auto generated via source code.
+
+## API
+
+Commandline is the easist way to extend Pouch's ability. And API extention will bring more powerful experience for users.
```
Comment (Letty5411) [inline docs/README.md]: easist -> easiest

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Commandline is the easist way to extend Pouch's ability. And API extention will bring more powerful experience for users. 
+
+### HTTP API
+
+Document [HTTP-API.md](api/HTTP_API.md) contains all raw HTTP API details. For Pouch's HTTP API, it is all recorded in file [swagger.yml](../apis/swagger.yml). The document HTTP_API.md is also auto generated by swagger.yml. So, we can still guarantee the strong consistency between code implementation and HTTP API document.
+
+### HTTP API Changelog
+
+Document [HTTP-API-CHANGELOG.md](api/HTTP_API_CHANGELOG.md) contains HTTP API changelog. In principle, HTTP API of Pouch should always keep still even if more versions are released. However, when pouch evolves, more functionality, direction adjustment and bugfix in API would make HTTP API change, and we pouch provides HTTP API changelog to track all the evloving.
+
```
Comment (Letty5411) [inline docs/README.md]: evloving -> envloving

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+[Test Doc](test) is the best reference helping contributors get aware of how to setup testing environment and do it. Currently we can divide test of Pouch into four dimensions: unit test, integration test of API and CLI, and CRI validation test. For more details, please refer to [test](test).
+
+## Contributions
+
+[Contributions](contributions) is a helpful document for community participants to learn how to contribute and coordinate within Pouch open source project. We draw up some friendly guidance for all participants.
+
+## Underlying Tech
+
+[Underlying Tech](underlying_tech) helps users to get more knowledge about the uderlying technology adopted in Pouch, such as cGroup, Namespce technology in Linux Kernel, hypervisor-based container technology, lxcfs, and so on. If users wish to master Pouch technology better, this document is a must-read.
```
Comment (Letty5411) [inline docs/README.md]: the uderlying technology -> underlying

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## Static Files
+
+[Static files](static_files) contains all static files for Pouch. If users are seeking some material in particular, such as Pouch log, architecture of it, or more pic about pouch. This document diectory is where users need a try.
```
Comment (Letty5411) [inline docs/README.md]: diectory -> directory

**Comment by Letty5411** [comment]
others LGTM

**Comment by allencloud** [comment]
Thanks @Letty5411

**Pull request merged**